app/src/main/java/org/fossasia/susi/ai/activities/SignUpActivity.java

```diff
@@ -100,28 +102,33 @@ public void onCancel(DialogInterface dialog) {
                 signUp.setEnabled(true);
             }
         });
+
         signUpCall.enqueue(new Callback<SignUpResponse>() {
             @Override
             public void onResponse(Call<SignUpResponse> call, Response<SignUpResponse> response) {
                 if (response.isSuccessful() && response.body() != null) {
                     Toast.makeText(SignUpActivity.this, response.body().getMessage(), Toast.LENGTH_SHORT).show();
+                    Intent intent = new Intent(SignUpActivity.this, LoginActivity.class);
+                    intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Why not just call finish() on the SignUpActivity?
The app will automatically go back to LoginActivity.
Also I don't think that this change is needed as it is already implemented in 
[https://github.com/fossasia/susi_android/pull/214/files#diff-003da66da67250f5d67cc94535536c8eR43](https://github.com/fossasia/susi_android/pull/214/files#diff-003da66da67250f5d67cc94535536c8eR43).